libqtile/widget/wlan.py

```diff
@@ -41,6 +43,28 @@ def get_status(interface_name):
     return essid, quality
 
 
+def get_private_ip(interface_name):
+    try:
+        result = subprocess.run(
+            ["ip", "-brief", "addr", "show", "dev", interface_name],
+            capture_output=True,
+            text=True,
+            check=True,
+        )
+    except subprocess.CalledProcessError:
```

Comment:
```suggestion
    except (subprocess.CalledProcessError, OSError):
```
Weirdly, the docs built fine on my fork of this. However, given that `subprocess.run` can raise an `OSError` too, we should catch that here.